src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -338,6 +339,13 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, kuberne
         cmd_helm_install.extend(["--kubeconfig", kube_config])
     if kube_context:
         cmd_helm_install.extend(["--kube-context", kube_context])
+
+    if least_privilege:
+        platformServiceAccountName = get_serviceaccount_name_from_configsettings(config_settings)
```

Comment:
shouldn't you pass auto upgrade as false here?

Reply:
actually we're blocking autoupdate for LP from agent side itself. But thanks for pointing out, would be good to keep config map up to date